src/empirical.jl

```diff
@@ -85,16 +85,17 @@ function EmpiricalVariogram(sdata, var₁::Symbol, var₂::Symbol=var₁;
 
   # choose accumulation algorithm
   if algo == :ball && isfloat && isminkowski
-    sums, counts = ball_search_accum(sdata, var₁, var₂, hmax, nlags, distance)
+    sums, cumdists, counts = ball_search_accum(sdata, var₁, var₂, hmax, nlags, distance)
```

Comment:
Could you please rename these variables to `xsums` and `ysums`. This way we can repeat the code for abscissa and ordinate:

```julia
abscissa = @. xsums / counts
ordinate = @. (ysums / counts) / 2
```